src/web-ui/src/public/Main.vue

```diff
@@ -2,7 +2,7 @@
   <Layout>
     <div class="content container">
       <RecommendedProductsSection
-        v-if="personalizeUserID"
+        v-if="user && personalizeUserID"
```

Comment:
The new cold user behavior does not require a user, only a `personalizeUserID`. Remove the `user` check here.

Reply:
Updated in #122